src/openrct2/entity/Peep.cpp

```diff
@@ -982,6 +988,8 @@ void Peep::Update()
         if (State == PeepState::Queuing)
             stepsToTake += stepsToTake / 2;
     }
+    if (stepsToTake < 55 && IsOnPathBlockedByVehicle())
+        stepsToTake = 55;
```

Comment:
Please introduce a constant for this magic value, even if this is the only place it is used. Named variables are so much easier to understand.

Reply:
```suggestion
    constexpr auto minStepsForCrossing = 55;
    if (stepsToTake < minStepsForCrossing && IsOnPathBlockedByVehicle())
        stepsToTake = minStepsForCrossing;
```

A comment stating why this block is here would also be nice.

Reply:
You're right, I added the constant and the comment.